types/react-color/lib/components/sketch/Sketch.d.ts

```diff
@@ -18,7 +18,7 @@ export interface SketchPickerStylesProps {
 
 export interface SketchPickerProps extends ColorPickerProps<SketchPicker> {
     disableAlpha?: boolean;
-    presetColors?: string[];
+    presetColors?: string[] | { name: string; color: string }[];
```

Comment:
From the docs page you linked:

> may also be described as an array of objects with color and title properties

Am I missing something, or should this be `title` instead of `name`?

Reply:
you're absolutely right! I guess I copied that incorrectly from my local test.